module/CourseInfo.cpp

```diff
@@ -0,0 +1,83 @@
+/**
+ *@file CourseInfo.cpp
+ *@brief コース情報
+ *@author higuchi/miyashita

```

Comment:
他の人のフォーマットと整合性を持たせるなら、`*`と`@`の間にスペースを入れたほうが良い気がします。
あと、ユーザ名の区切り文字も。

```suggestion
 * @file CourseInfo.cpp
 * @brief コース情報
 * @author higuchi,miyashita

```

@arimakaoru @shigyo123 @mutotaka0426 
流石に、この部分は各自で好きなように書く方針のほうが良いですか？

(個人的には、ユーザ名の区切り文字の後にスペースを１つ入れるようにしたい...)

Reply:
まあ、揃ってると綺麗だよね